data/campaigns/Heir_To_The_Throne/scenarios/05b_Isle_of_the_Damned.cfg

```diff
@@ -341,60 +341,58 @@
             message= _ "Surprise! Searching for magi, and all I get is these foul humans!"
         [/message]
     [/event]
+#endif
 #enddef
 
     [event]
         name=prestart
 
-        #
-        # Determine the contents of the temples:
-        # - on easy, Moremirmu shows up in 1 close temple and the others are empty
-        # - on normal, Moremirmu shows up in 1 close temple and the far one has the revenant
-        # - on hard, Moremirmu and the revenant show up in the 2 close temples (far one is empty)
-        #
-        {RANDOM 1..2}
-        [if]
-            [variable]
-                name=random
-                numerical_equals=1
-            [/variable]
-#ifdef EASY
-            [then]
+        {RANDOM 1..6}
```

Comment:
They were never true.

**OLD WAY**
On EASY there was no Revenant, so no issue. So check the closest and, if empty, check the middle. That's 50/50 odds to find Moremirmu, and 0% to find the Revenant. The only randomness was which temple had Moremirmu and which was empty.

On NORMAL, the Revenant was in the furthest temple. So check the check the closest and if he's not there, check the middle.  The odds are 50/50 for Moremirmu. The odds of finding the Revenant at the furthest temple are 100%. The only randomness was which temple had Moremirmu and which was empty, just as with EASY.

On HARD, the Revenent and Moremiru are both in the closest two temples. The odds of the farthest temple being the empty temple are 100%. The odds of finding Moremirmu in the closes temple was 50% .. if it was not Moremirmu it was the Revenant.

So, yes, it's harder. On EASY it's trivially harder. On NORMAL it's infinitely harder (because there was no chance of accidentally finding the Revenant).

But on HARD it's easier because the odds of hitting the Revenant on your first choice are now 1-in-3 instead of 1-in-2.